show/main.py

```diff
@@ -455,6 +465,14 @@ def syseeprom():
     run_command("sudo decode-syseeprom")
 
 
+# 'hardware' command ("show hardware")
+@cli.command()
+def hardware():
+    """Show local hardware information"""
+    command = "sudo dmidecode -t 1,2,10"
```

Comment:
do we have this dmidecode in the base image? also, which dmidecode is related to hardware. This is not requirement for platform support, there is no guarantee that this will be supported by all sonic platforms.

Reply:
No, it's not in base-image, i was planning to add it though. However, i just noticed that you guys recently introduced "show platform", which overlaps with the info provided by this command, so i will remove this one from my patch.